main.go

```diff
@@ -83,7 +88,8 @@ Examples:
 }
 
 func makeTrivyJsonReport(outputFileName string) error {
-	trivyCommand := os.Args[1 : len(os.Args)-1]
+	trivyCommandBorder := getFirstFlagIndex()
+	trivyCommand := os.Args[1:trivyCommandBorder]
```

Comment:
Go functions can return several values:
```sh
def getParams() (string, strign, []string) {}
```
or we can create a new struct:
```sh
type pluginFlags struct {
  htmlFile string
  inputFile string
}
// and return it
def getFlags() (*pluginFlags, []string) {
}
```
`[]string` - is new arguments for Trivy,

Wdyt?